pilot/util/harvester.py

```diff
@@ -267,6 +294,7 @@ def publish_job_report(job, args, job_report_file="jobReport.json"):
     :param args: Pilot arguments object.
     :param job_report_file: name of job report (string).
     :raises FileHandlingFailure: in case of IOError.
+    :return True or False
     """
 
     src_file = join(job.workdir, job_report_file)
```

Comment:
did you mean os.path.join?

Reply:
yes it works with the import but will change for readability